alerts/openshift-virtualization-operator/PodSchedulingMightBeImbalancedAcrossNodes.md

```diff
@@ -0,0 +1,22 @@
+# PodSchedulingMightBeImbalancedAcrossNodes
+
+## Meaning
+
+This alert is applicable to clusters that have a large number of container
+images per node. It is triggered when a node exceeds the maximum number of
+configured images (default: 50). Once this limit is reached, the scheduler will
```

Comment:
"scheduler will be unable" - Change to "scheduler is unable...".
Otherwise the tenses will conflict (present tense in subordinate clause, future tense in principle clause).

Also, change "any new pods" to "new pods". "any" is redundant.

Reply:
fixed